src/Symfony/Bundle/WebProfilerBundle/Tests/Resources/IconTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\Tests\Resources;
+
+use Symfony\Bundle\WebProfilerBundle\Tests\TestCase;
+
+class IconTest extends TestCase
+{
+    public function testIconFileContents()
+    {
+        $iconFiles = glob(__DIR__.'/../../Resources/views/Icon/*.svg');
```

Comment:
I suggest moving this glob to a data provider instead, so that all icon files are checked when running tests, instead of reporting only the first failed file